### PR TITLE
bump: update golang to 1.17 for ci-search

### PR DESCRIPTION
**Pull request by eggfoobar**
Signed-off-by: ehila <ehila@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift/ci-search/openshift-ci-search-master.yaml [hunk 1]
```diff
@@ -3,21 +3,21 @@ base_images:
...
-    tag: golang-1.13
+    tag: golang-1.17
 images:
 - from: base
   inputs:
-    openshift_release_golang-1.13:
+    openshift_release_golang-1.17:
       as:
-      - registry.svc.ci.openshift.org/openshift/release:golang-1.13
+      - registry.svc.ci.openshift.org/openshift/release:golang-1.17
```
Comment (bradmwilliams) [inline ci-operator/config/openshift/ci-search/openshift-ci-search-master.yaml:20]: This needs to be: registry.ci.openshift.org/openshift/release:golang-1.17

**Review by bradmwilliams — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/ci-search/openshift-ci-search-master.yaml (hunk 1 above)
Comment (eggfoobar) [reply ci-operator/config/openshift/ci-search/openshift-ci-search-master.yaml]: Sounds good, updated

**Pull request merged**